driver/driver.go

```diff
@@ -294,19 +319,19 @@ func isContain(obj string, target []string) bool {
 	return false
 }
 
-func generateCustomiseConfig(generateDir, hostDir string, opts Options) error {
+func generateCustomiseConfig(podUID, hostDir string, opts Options) error {
 	var err error
-	outputProjectPath := path.Join(svcProjectLogConfigDir, generateDir+".conf")
-	outputClusterPath := path.Join(svcClusterLogConfigDir, generateDir+".conf")
+	configFileName := fmt.Sprintf("%s_%s.conf", podUID, opts.VolumeName)
+	outputProjectPath := path.Join(svcProjectLogConfigDir, configFileName)
+	outputClusterPath := path.Join(svcClusterLogConfigDir, configFileName)
 	conf := map[string]interface{}{
 		"Format":         opts.Format,
 		"Path":           fmt.Sprintf("%s/*.*", hostDir),
-		"Project":        fmt.Sprintf("%s:%s", opts.ClusterID, opts.ProjectID),
-		"ClusterPosPath": fmt.Sprintf("/fluentd/log/custom_cluster_userformat_%s.pos", generateDir),
-		"ProjectPosPath": fmt.Sprintf("/fluentd/log/custom_project_userformat_%s.pos", generateDir),
+		"ClusterPosPath": fmt.Sprintf("/fluentd/log/custom_cluster_userformat_%s_%s.pos", podUID, opts.VolumeName),
+		"ProjectPosPath": fmt.Sprintf("/fluentd/log/custom_project_userformat_%s_%s.pos", podUID, opts.VolumeName),
 	}
 
-	tmpClusterConfigFile := path.Join(tmpClusterDir, generateDir+".conf")
+	tmpClusterConfigFile := path.Join(tmpClusterDir, configFileName)
```

Comment:
I think both old and new config will be applied in fluentd. So we will send two logs to log target after  update.